scripts/patch-epinio-deployment.sh

```diff
@@ -60,7 +66,19 @@ echo "Waiting for dummy pod to be ready"
 kubectl wait --for=condition=ready pod -n epinio epinio-copier
 
 echo "Copying the binary on the PVC"
-kubectl cp ${EPINIO_BINARY_PATH} epinio/epinio-copier:/epinio/epinio
+# Notes
+# 1. kubectl cp breaks because of the colon in `pod:path`. Thus the more complex tar construction.
```

Comment:
Will probably be fixed at some point: https://github.com/kubernetes/kubernetes/pull/102047